live-examples/css-examples/lists/counter-reset.html

```diff
@@ -0,0 +1,41 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="list-style">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">counter-reset: chapter-count 0;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+                <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
```

Comment:
I think it would be good to have (possibly as the first option and selected initially), an extra option with `counter-reset: none;` so we can see the initial effect without any `counter-reset` happening.

What do you think?

Reply:
Yeah, this sounds like a good idea. Adding it now.